### ISSUE TITLE
Compilation error (maybe some conflict with `elm-lazy-list`)

### ISSUE THREAD

**Issue by zdychacek** [jinjor/elm-time-travel#6]
Hi again,

after I have added `elm-time-travel` to my project's dependencies, I am experiencing an issue related with using a `elm-lazy-list`.

Compilation error:

"The elm-package.json constraints of 'elm-community/elm-lazy-list' are probably
letting too much stuff through. Definitely open an issue on the relevant github
repo to get this fixed and save other people from this pain."

Actually, I don't know if this issue is related with `elm-time-travel` or with `elm-lazy-list`...

**Comment by jinjor** [comment]
Hmm... I have seen this error before but I forgot what it was caused by. 

Can you show your project's dependencies here?

**Comment by zdychacek** [comment]
My dependencies are:

```
"elm-community/json-extra": "1.0.0 <= v < 2.0.0",
"elm-community/maybe-extra": "1.1.0 <= v < 2.0.0",
"elm-lang/core": "4.0.0 <= v < 5.0.0",
"elm-lang/html": "1.0.0 <= v < 2.0.0",
"elm-lang/navigation": "1.0.0 <= v < 2.0.0",
"evancz/elm-http": "3.0.1 <= v < 4.0.0",
"evancz/elm-markdown": "3.0.0 <= v < 4.0.0",
"krisajenkins/elm-exts": "25.6.2 <= v < 26.0.0",
"rgrempel/elm-route-url": "2.0.1 <= v < 3.0.0"
```

**Comment by jinjor** [comment]
I tried to find a clue, but was not able to. Sorry for inconvenience.

**Comment by zachwilliamson91** [comment]
Could be perhaps related to https://github.com/elm-lang/elm-compiler/issues/1430

**Comment by jvoigtlaender** [comment]
See https://github.com/elm-lang/elm-compiler/issues/1430#issuecomment-229932911.

**Comment by jinjor** [comment]
Thank you for your information!

So using infix operator is problematic? Good to know.
I'll update the dependency as soon as it is fixed. (Or, I use another library if there is one)

**Comment by jvoigtlaender** [comment]
Yes, infix operators are problematic. Both due to that compiler issue and due to this advice against them: http://package.elm-lang.org/help/design-guidelines#avoid-infix-operators.

**Comment by jinjor** [comment]
To all who are blocked on this,

I switched parser from `Dandandan/parser` to `Bogdanp/elm-combine` for another reason (= performance) and just published new version (1.0.11). If your problem has been solved by this change, please let me know.

**Comment by zdychacek** [comment]
Many thanks!! It works!
